packages/naming.cell.match/cell-match.test.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const safeEval = require('node-eval');
+
+const BemCell = require('@bem/sdk.cell');
+const { origin } = require('@bem/sdk.naming.presets');
```

Comment:
We use destructuring assignment but we aware of spread in object expression (using `Object.assgin`). Why?

Reply:
"package.json" declares node 4 and it has no support of destructuring http://node.green/#ES2015-syntax-destructuring--assignment

Reply:
Well, I guess if we need node 4 then we need to fix it.

I'm just living in the future ;-(

Reply:
We decided to drop 4.x support so it can be left as is.